website/docs/r/oracledatabase_exadata_infrastructure.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Oracle Database"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_oracle_exadata_infrastructure"
+description: |-
+  Manages a Exadata Infrastructure.
```

Comment:
Before I continue leaving suggestions around consistency for the resource docs, I don't mind if it's referred to as `Cloud Exadata Infrastructure` or `Exadata Infrastructure`, but can the naming as well as spacing and casing be consistent across the resource and data source documentation?

Reply:
Let's go with `Cloud Exadata Infrastructure`.